extensions/ql-vscode/test/unit-tests/databases/config/db-config-store.test.ts

```diff
@@ -184,6 +185,146 @@ describe("db config store", () => {
     });
   });
 
+  describe("add db items", () => {
```

Comment:
We could follow the same pattern in naming as the other tests e.g. 
```suggestion
  describe("db and list addition", () => {
```

Reply:
👍